app/views/users/show.html.erb

```diff
@@ -1 +1,24 @@
-<h2>One user and posts will be displayed here</h2>
+<div class="single-user-container">
+<div class="image-and-name-container">
+<img src="<%= @user.photo %>" />
+<h2><%= @user.name %></h2>
+<p>Number of Posts: <%= @user.postscounter %></p>
+</div>
+
+<div class="user-bio">
+<h4>Bio</div>
+<p><%= @user.bio %></p>
+</div>
+
+<div class="user-posts">
+<%@posts.each do |post| %>
+<h4>Post #<%= post.id %></h4>
+<h4><%= post.text %></h4>
+<p>Comments: <%post.commentscounter%></p>
+<p>Likes: <%post.likescounter%></p>
+<%end%>
```

Comment:
- [ ] the application should only display the recent posts which you can do by using the `show_recent_posts` method you created in the user model. then you don't have to define `@posts` in the user_controller `show` action.
- [ ] there should be a button to show all the posts. which redirects to the user_post_path.